__tests__/app.test.js

```diff
@@ -75,4 +75,53 @@ describe("GET /api/reviews/:review_id", () => {
             expect(response.body.msg).toBe("bad request");
         })
     });
-})
\ No newline at end of file
+});
+
+describe("PATCH /api/reviews/:review_id", () => {
+    test('200: responds with an updated review depending on inc_votes', () => {
+        return request(app)
+        .patch("/api/reviews/1")
+        .send({ inc_votes: 3})
+        .expect(200)
+        .then((response) => {
+            expect(response.body.review).toEqual({
```

Comment:
I would change this toEqual to toMatchObject, this will future proof your test when we have to add comment_count later